sdk/schemaregistry/azure-data-schemaregistry-avro/src/main/java/com/azure/data/schemaregistry/avro/SchemaRegistryAvroDeserializerBuilder.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.schemaregistry.avro;
+
+import com.azure.core.credential.TokenCredential;
+
+/**
+ * Builder class for constructing {@link SchemaRegistryAvroDeserializer} and {@link SchemaRegistryAvroAsyncDeserializer}
+ */
+public class SchemaRegistryAvroDeserializerBuilder {
+
+    private final String registryUrl;
+    private TokenCredential credential;
+    private boolean avroSpecificReader;
+    private Integer maxSchemaMapSize;
+
+    /**
+     * Instantiates instance of Builder class.
+     * Supplies client defaults.
+     *
+     * @param registryUrl base schema registry URL for storing and fetching schemas
+     */
+    public SchemaRegistryAvroDeserializerBuilder(String registryUrl) {
```

Comment:
Consider using no-args ctor for builder and have a setter method for registryUrl

Reply:
Updated both serializer and deserializer builders to have separate schemaRegistryUrl() methods.